src/lib/bind/accessors.html

```diff
@@ -174,7 +174,8 @@
       // ReadOnly properties have a private setter only
       // TODO(kschaaf): Per current Bind factoring, we shouldn't
       // be interrogating the prototype here
-      if (model.getPropertyInfo && model.getPropertyInfo(property).readOnly) {
+      var info = model.getPropertyInfo(property);
```

Comment:
No longer guarding against model not having `getPropertyInfo`. We still need to do this.